api_specs/v2/component_analyses.yaml

```diff
@@ -269,6 +311,92 @@ components:
+                  url: "https://snyk.io/vuln/SNYK-PYTHON-DJANGO-40055"
+                  cve_ids: [
+                    "CVE-2010-4534"
+                  ]
+                  fixed_in:
+                    - "1.1.3"
+                    - "1.2.4"
+                - id: "SNYK-2010-30812"
+                  cvss: "1.2"
+                  is_private: false
+                  cwes:
+                    - "CWE-264"
+                  cvss_v3: "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N"
+                  severity: "medium"
+                  title: "Information Exposure"
+                  url: "https://snyk.io/vuln/SNYK-PYTHON-DJANGO-40055"
+                  cve_ids: [
+                    "CVE-2010-4534"
+                  ]
+                  fixed_in:
+                    - "1.1.3"
+                    - "1.2.4"
+              recommended_versions : "2"
+              message: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+              highest_severity: "high"
+              known_security_vulnerability_count: 2
+              security_advisory_count: 1
+              exploitable_vulnerabilities_count: 0
+              vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
+
+            - package: "django/pck2"
+              version: "1.1"
+              package_unknown: false
+              vulnerabilities:
+                - id: "SNYK-2010-3082"
+                  cvss: "1.2"
+                  is_private: false
+                  cwes:
+                    - "CWE-264"
+                  cvss_v3: "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N"
+                  severity: "medium"
+                  title: "Information Exposure"
+                  url: "https://snyk.io/vuln/SNYK-PYTHON-DJANGO-40055"
+                  cve_ids: [
+                    "CVE-2010-4534"
+                  ]
+                  fixed_in:
+                    - "1.1.3"
+                    - "1.2.4"
+                - id: "SNYK-2010-30812"
+                  cvss: "1.2"
+                  is_private: false
+                  cwes:
+                    - "CWE-264"
+                  cvss_v3: "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N"
+                  severity: "medium"
+                  title: "Information Exposure"
+                  url: "https://snyk.io/vuln/SNYK-PYTHON-DJANGO-40055"
+                  cve_ids: [
+                    "CVE-2010-4534"
+                  ]
+                  fixed_in:
+                    - "1.1.3"
+                    - "1.2.4"
+              recommended_versions : "2"
+              message: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+              highest_severity: "high"
+              known_security_vulnerability_count: 2
+              security_advisory_count: 1
+              exploitable_vulnerabilities_count: 0
+              vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
```

Comment:
vendor_package_link and recommended_versions would be have same value for all root & included_packages values right?